maestro-orchestra/src/main/java/maestro/orchestra/yaml/YamlSleepCommand.kt

```diff
@@ -0,0 +1,14 @@
+package maestro.orchestra.yaml
+import com.fasterxml.jackson.annotation.JsonCreator
+data class YamlSleepCommand(
+    val time: Long?,
```

Comment:
Is there a better label for this?
Should this be in milliseconds? Is there a good use case for something more granular than seconds?

Would this be more readable?
```
- sleep
    seconds: 5
```

(that's intended as a real question, not a leading question)